src/datacatalog/plugins/postgres/__init__.py

```diff
@@ -122,6 +133,7 @@
     while connect_attempt_tries_left >= 0:
         try:
             await app['pool'].execute(_Q_CREATE)
+            await app['pool'].execute(_Q_CREATE_STARTUP_ACTIONS)
```

Comment:
Waarom deze aparte `_Q_CREATE_STARTUP_ACTIONS`, in plaats van extra `CREATE`-statements toevoegen aan `_Q_CREATE`?